include/jsFunctions.php

```diff
@@ -0,0 +1,41 @@
+<?php
+include('config/config.php');
+include('config/init.php');
+ ?>
+ 
+<script type='text/javascript'>
+
+function upVoteNumber($myEventVotes, $eventID){
```

Comment:
I see you've already done part of what I suggested above. But I'll point out: this function takes in two parameters, but you haven't actually passed these parameters to the function when calling it.

Reply:
Also, I'd try to figure out a way to do this where you don't need to pass `$myEventVotes` around like this. It seems a bit clunky. Let me know if you want to talk about ideas for improving that (but I bet you can figure it out).